pkg/alertmanager/amcfg.go

```diff
@@ -1470,6 +1525,15 @@ func (sc *snsConfig) sanitize(amVersion semver.Version, logger log.Logger) error
 	return sc.HTTPConfig.sanitize(amVersion, logger)
 }
 
+func (tc *telegramConfig) sanitize(amVersion semver.Version, logger log.Logger) error {
+	telegramAllowed := amVersion.GTE(semver.MustParse("0.24.0"))
+	if !telegramAllowed {
+		return fmt.Errorf(`invalid syntax in receivers config; telegram integration is available in Alertmanager >= 0.24.0`)
+	}
+
```

Comment:
we need to check ChatID and BotToken too.

Reply:
Done!